sdk/tests/deqp/functional/gles3/es3fFboMultisampleTests.js

```diff
@@ -305,13 +305,14 @@ var DE_ASSERT = function(x) {
             gl.R8,
 
             // gl.EXT_color_buffer_float
-            gl.RGBA32F,
-            gl.RGBA16F,
-            gl.R11F_G11F_B10F,
-            gl.RG32F,
-            gl.RG16F,
-            gl.R32F,
-            gl.R16F
+            // Multi-sample floating-point color buffers are not supported, see https://www.khronos.org/registry/webgl/extensions/EXT_color_buffer_float/
```

Comment:
Also see the discussion here for chromium project: https://codereview.chromium.org/1437293002/patch/40001/50012